routes/api.js

```diff
@@ -60,5 +57,14 @@ export default router => {
             });
     });
 
-
+    router.post("/createPlaylist", (req, res) => {
+        spotifyApi.createPlaylist(req.body.userId, req.body.name, { public : false }).then(
+            function(data) {
+                res.json(data);
+            },
+            function(err) {
+                console.log('Something went wrong! (get playlists)', err);
+                res.status(err.statusCode).end();
```

Comment:
Does Spotify API provide useful information in response in case of a failure to create a playlist? Would be useful if we let the user know why it failed.

Reply:
Yeah I haven't done any error handling yet, need to account for if the server is down or if authentication isn't working, etc, I'll make an issue about it soon.